embedding/embedlite/embedshared/EmbedLitePuppetWidget.cpp

```diff
@@ -561,6 +562,21 @@ EmbedLitePuppetWidget::GetLayerManager(PLayerTransactionChild* aShadowManager,
   return mLayerManager;
 }
 
+float
+EmbedLitePuppetWidget::GetDPI()
+{
+  if (mDPI < 0) {
+    if (mEmbed) {
```

Comment:
With this change the PR won't compile in the `embedlite` branch, because @tworaz's changes are missing here. I'll push this PR to `embedlite_38` and prepare a PR to `embedlite` with Piotr's commits.